docs/source/management/index.md

```diff
@@ -108,16 +112,104 @@ Available SRE configurations for context 'green':
 └──────────────┴──────────┘
 ```
 
-## Removing a deployed Data Safe Haven
+### Removing a deployed Data Safe Haven
 
 - Run the following if you want to teardown a deployed SRE:
 
 ```{code} shell
 $ dsh sre teardown YOUR_SRE_NAME
 ```
 
+::::{admonition} Tearing down an SRE deletes ALL data and resources
+Running `dsh sre teardown` will remove all resources deployed within the SRE.
+Ensure that any desired outputs have been egressed before deleting the SRE.
+Any data remaining on the SRE will be deleted.
+The user groups for the SRE on Microsoft Entra ID will also be deleted.
+::::
```

Comment:
```suggestion
::::{danger} Tearing down an SRE is destructive and irreversible 
Running `dsh sre teardown` will destroy **all** resources deployed within the SRE.
Ensure that any desired outputs have been extracted before deleting the SRE.
**All** data remaining on the SRE will be deleted.
The user groups for the SRE on Microsoft Entra ID will also be deleted.
::::
```

Some suggestions for changing the wording and emphasis. (And the classic, egress is not a verb).